content/release-notes/2022-10-25.md

```diff
@@ -0,0 +1,16 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Compute: Add support of PostgreSQL 15.0 and its PostgreSQL extensions.
+For information about supported extensions, see [Available PostgreSQL extensions](https://neon.tech/docs/reference/compatibility/#available-postgresql-extensions).
+- Compute: Fix creation of databases with trailing spaces in database name.
+- Compute: Disable `wal_log_hints` by default.
+- Pageserver: Add timeline state field to TimelineInfo (returned by `timelines` internal management API endpoint).
+- Pageserer: Merge 'local' and 'remote' parts of TimelineInfo into one struct in the internal management API.
```

Comment:
Is there any functional change or is this just refactoring for better / cleaner code?

Reply:
I guess it's just a refactoring.
We don't expose this internal API anywhere, so we can remove this.

Reply:
OK. Let's remove.